po/ar.po

```diff
@@ -0,0 +1,2520 @@
+msgstr "اختيار"
+
+#. Label displayed in the keyboard shortcuts dialog
+#: src/ui/shortcuts.ui
+msgid "Delete the selection"
+msgstr "حذف التحديد"
+
+#. Label displayed in the keyboard shortcuts dialog
+#: src/ui/shortcuts.ui
+msgid "Import a file as the selection"
+msgstr "استيراد ملف كتحديد"
+
+#. Label displayed in the keyboard shortcuts dialog
+#: src/ui/shortcuts.ui
+msgid "Edit the main color (left click)"
+msgstr "تعديل اللون الرئيسي (left click)"
+
+#. Label displayed in the keyboard shortcuts dialog
+#: src/ui/shortcuts.ui
+msgid "Edit the secondary color (right click)"
+msgstr "تعديل اللون الثانوي (right click)"
+
+#: src/ui/shortcuts.ui
+msgid "History"
+msgstr "تاريخ"
+
+#. Context: this section of the shortcuts window is about moving
+#. the view to the left/right/top/bottom
+#: src/ui/shortcuts.ui
+msgid "Navigation"
+msgstr "ملاحة"
+
+#: src/ui/shortcuts.ui
+msgid "Tabs"
+msgstr ""
+
+#: src/ui/shortcuts.ui
+msgid "Toggle fullscreen mode"
+msgstr ""
+
+#: src/ui/shortcuts.ui
+msgid "Toggle toolbars visibility"
+msgstr "تبديل رؤية أشرطة الأدوات"
+
+#: src/ui/shortcuts.ui
+msgid "Touch gestures"
+msgstr "إيماءات اللمس"
+
+#: src/ui/window.ui src/deco_manager.py
+msgid "Loading…"
+msgstr "تحميل…"
+
+#: src/ui/window.ui
+msgid "Exit fullscreen"
+msgstr ""
+
+#: src/ui/win-menus.ui
+msgid "What's new in Drawing"
+msgstr "ما هو جديد في رسم"
+
+#: src/ui/win-menus.ui
+msgid "Dismiss"
+msgstr "الفصل"
+
+#: src/deco_manager.py
+#, python-format
+msgid "Undo %s"
+msgstr "تراجع %"
+
+#: src/deco_manager.py
+#, python-format
+msgid "Redo %s"
+msgstr "إعادة %s"
+
+#. Context: an error message
+#: src/history_manager.py
+msgid "Attempt to save an invalid state"
+msgstr "محاولة حفظ حالة غير صالحة"
+
+#: src/history_manager.py
+#, python-format
+msgid "Error: no tool '%s'"
+msgstr ""
+
+#. Context: an error message
+#: src/image.py
+#, python-format
+msgid "New pixbuf empty, no change applied to %s"
+msgstr ""
+
+#: src/image.py src/properties.py
+msgid "Unsaved file"
+msgstr "ملف غير محفوظ"
+
+#. Context: this is a debug information that users will never see
+#: src/image.py
+#, python-format
+msgid "%s frames per second"
+msgstr ""
+
+#. Description of a command line option
+#: src/main.py
+msgid "Show the app version"
+msgstr "إظهار إصدار البرنامج"
+
+#. Description of a command line option
+#: src/main.py
+msgid "Open a new window"
+msgstr "افتح نافذة جديدة"
+
+#. Description of a command line option
+#: src/main.py
+msgid "Open a new tab"
+msgstr ""
+
+#. Description of a command line option
+#: src/main.py
+msgid "Edit the clipboard content"
+msgstr "تعديل محتوى الحافظة"
+
+#: src/main.py
+msgid "This version isn't stable!"
+msgstr "هذا الإصدار غير مستقر!"
+
+#: src/main.py
+msgid "Report bugs or ideas"
+msgstr "الإبلاغ عن الأخطاء أو الأفكار"
+
+#. To tranlators: "translate" this by a list of your names (one name
+#. per line), they will be displayed in the "about" dialog
+#: src/main.py
+msgid "translator-credits"
+msgstr "المترجمين"
```

Comment:
i just noticed this. As suggested by the comment, it was supposed to be the name under which you want to be credited

i'm putting your github account name, but if you want something else please tell me

Reply:
I would prefer my Codeberg account. https://codeberg.org/contractover
but if you prefer to put my GitHub, that's fine too.

Reply:
it's a name or a pseudonym, not a link to an actual account, for now i've written "@contractisover"